individual-files/facilities.c

```diff
@@ -0,0 +1,28 @@
+#include <stdio.h>
+#include <string.h>
+
+char *sequence = "class department college office field library";
+void unique( const char *list, char *buf )
+{
+  char name[100];
+  int k;
+  buf[0] = 0;
+  for(;;)
```

Comment:
Not sure if this is a bad practice, but I'm more use to seeing while(1) to implement an infinite loop.